### ISSUE TITLE
Variable Declaration 

### ISSUE THREAD

**Issue by uittorio** [uittorio/ts-auto-mock#89]
In typescript you can define a Variable with the same name as an interface
```ts
var Test: Test;

interface Test {
   a: string;
}

createMock<Test>();
```
This should refere to Test interface and not variable. 

Typescript should always use the interface and not the variable.

**Issue**
The compile fail because its trying to use variable declaration as type;
TsAutoMock doesn't distinguish about multiple declarations and will always use the first one.

Possible solution
A possible solution is to ignore any variable declaration from declarations list.

**Comment by uittorio** [comment]
#83

**Comment by uittorio** [comment]
This issue has been resolved in version 1.3.1
